glossary.yml

```
@@ -3285,6 +3285,15 @@
     def: >
       The statement or statements executed by a loop.
 
+- slug: lsof
+  en:
+    term: "lsof"
+    def: >
+      Linux command to see the list of open files being used by processes. 
```

```suggestion
      UNIX command to see the list of open files being used by [processes](#process). 
```

Add internal link. This command also exists in Mac Terminal, so is not Linux-specific.

---

You are right! Thanks!